### ISSUE TITLE
Provide original hammer events to the user

### ISSUE THREAD

**Issue by mashuai** [almende/vis#1906]
When I use 'rangechange' ,'rangechanged','timechange' or 'timechanged', I want to get the hammer event object like I want to use `pannleft` or `panright`, sometime I need the delta. I have change the code like

``` javascript
 this.body.emitter.emit('rangechange', {
      start: startDate,
      end: endDate,
      byUser: true,
      originEvent: event
    });
```

Will vis add this feature?

**Comment by yotamberk** [comment]
Sounds like a good idea!
Can you submit a PR with this addition?

**Comment by mojoaxel** [comment]
@yotamberk Letzts change this issue to a feature-request.

**Comment by yotamberk** [comment]
closed with #2420
